src/ec/eddsa.rs

```diff
@@ -196,8 +198,12 @@ struct Point {
 type Elem = [i32; ELEM_LIMBS];
 const ELEM_LIMBS: usize = 10;
 
-type PublicKey = [u8; ELEM_LEN];
+type PublicKey = [u8; PUBLIC_KEY_LEN];
+const PUBLIC_KEY_LEN: usize = ELEM_LEN;
 
+const PRIVATE_KEY_LEN: usize = ELEM_LEN;
+const KEY_PAIR_LEN: usize = PUBLIC_KEY_LEN + PRIVATE_KEY_LEN;
+const SIGNATURE_LEN: usize = 64;
```

Comment:
The signature is, IIUC, one element concatenated with one scalar, so `ELEM_LEN + SCALAR_LEN`.